.github/workflows/build.yml

```diff
@@ -93,16 +93,16 @@ jobs:
           distribution: temurin
           cache: 'maven'
           server-id: ossrh
-          server-username: OSSRH_JIRA_USERNAME
-          server-password: OSSRH_JIRA_PASSWORD
+          server-username: SONATYPE_USERNAME
+          server-password: SONATYPE_PASSWORD
           gpg-private-key: ${{ secrets.SIGNING_KEY }}
           gpg-passphrase: GPG_PASSPHRASE
       - name: Deploy artifacts to Maven Central
         run: |
           ./mvnw clean deploy -B -e -P deploy -DskipTests
         env:
-          OSSRH_JIRA_USERNAME: eller86
-          OSSRH_JIRA_PASSWORD: ${{ secrets.OSSRH_JIRA_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
@gtoison In your ask, you asked for SONATYPE_TOKEN which better aligned so I added it as token.  Can you change to SONATYPE_TOKEN?

Reply:
Ah I just saw your message, that was indeed a mistake on my part